lib/chef/knife/ec_restore.rb

```diff
@@ -238,11 +238,17 @@ def upload_org_data(name)
           # pivotal and members of the admins group. Use the same strategy
           # above here.
           #
-          ['admins', 'billing-admins', 'public_key_read_access'].each do |group|
+          groups = ['admins', 'billing-admins']
+          groups.push('public_key_read_access') if server.supports_public_key_read_access?
+
+          groups.each do |group|
             restore_group(chef_fs_config, group, :clients => false)
           end
 
-          ['/acls/groups/billing-admins.json', '/acls/groups/public_key_read_access.json'].each do |acl|
+          acls_groups_paths = ['/acls/groups/billing-admins.json']
+          acls_groups_paths.push('/acls/groups/public_key_read_access.json') if server.supports_public_key_read_access?
```

Comment:
@stevendanna high-level thoughts on how to handle that?  Address here or in a new PR?